src/ess/dream/io.py

```diff
@@ -0,0 +1,122 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+import os
+from typing import Any, Dict, Union
+
+import scipp as sc
+import scippnexus as snx
+
+
+def load_nexus(
+    path: Union[str, os.PathLike],
+    *,
+    load_pixel_shape: bool = False,
+    entry: str = 'entry',
+    fold_detectors: bool = True,
+) -> sc.DataGroup:
+    """
+    Load an unprocessed DREAM NeXus file.
+
+    See https://confluence.esss.lu.se/pages/viewpage.action?pageId=462000005
+    and the ICD DREAM interface specification for details.
```

Comment:
This isn't great. Are there publicly accessible resources about this? Can we make these public?

Reply:
Why public? This is an ESS repo, so I think everyone who needs access can get to the info?

Reply:
I don't know how that will work for visitors. But fine.